docs/decisions/0001-namespacing-branches.rst

```diff
@@ -0,0 +1,71 @@
+Namespacing Branches and Tags
+-----------------------------
```

Comment:
The preferred order of headings for new Open edX rst docs is `#`, `*`, `=`, `-` ("four lines, three lines, two lines, one line"). https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html?highlight=rst#headings

Reply:
It's unfortunately we have many exemplars around our repos that don't reflect the preferred order.  I assume we don't have any automation for this?